api/client/batch_client.py

```diff
@@ -122,8 +122,10 @@ def log_request(start_time, retry_count, msg, status_code):
                     if retry_count > 0:
                         time.sleep(2 ** retry_count)
                     continue
-                elif status_code in [400, 401, 402, 404]:
+                elif status_code in [400, 401, 402, 404, 502]:
```

Comment:
502 should be retryable. Could put in the other list of status codes above

Reply:
Shoult be done now

Reply:
moved this change to https://github.com/gro-intelligence/api-client/pull/285